packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/DatacentreGeneralInformation.spec.tsx

```diff
@@ -5,16 +5,16 @@ import {
   datacentreList,
 } from '@ovh-ux/manager-module-vcd-api';
 import {
-  checkModalError,
-  checkModalVisibility,
+  checkModal,
+  checkModalText,
+  waitForOptions,
+} from '@ovh-ux/manager-core-test-utils';
+import {
   labels,
   mockEditInputValue,
   renderTest,
-} from '../../../../test-utils';
-import {
-  DEFAULT_TIMEOUT,
   mockSubmitNewValue,
-} from '../../../../test-utils/uiTestHelpers';
+} from '../../../../test-utils';
```

Comment:
tsconfig path does not work on this folder? e.g. `@/test-utils`

Reply:
my IDE displays an error but the test runs and succeed so I will change it